packages/redux/src/OidcSecure.tsx

```diff
@@ -71,12 +69,13 @@ const defaultPropsOidcSecure = {
 
 type OidcSecureProps = PropsWithChildren<{
   isEnabled?: boolean;
+  authenticatingComponent: ComponentType;
 }>;
 
 const OidcSecure: FC<OidcSecureProps> = props => {
-  const { isEnabled, children } = props;
+  const { isEnabled, children, ...configProps } = props;
   if (isEnabled) {
-    return <Secure>{children}</Secure>;
+    return <Secure {...configProps}>{children}</Secure>;
   }
```

Comment:
Yeah, awesome pr!
May be you should here just pass authenticatingComponent instead of a props spread

Reply:
I think it's better to leave spread Props to be more scalable, in general all inject props from parents should be spread, even in this case there's only one i agree, but may be in the future we can inject others without modifying this line